src/Sylius/Bundle/UserBundle/Security/UserPasswordEncoder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Security;
+
+use Sylius\Component\User\Model\UserInterface;
+use Sylius\Component\User\Security\UserPasswordEncoderInterface;
+use Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface;
+
+/**
+ * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
+ */
+class UserPasswordEncoder implements UserPasswordEncoderInterface
+{
+    /**
+     * @var EncoderFactoryInterface
+     */
+    private $encoderFactory;
+
+    /**
+     * @param EncoderFactoryInterface $encoderFactory
+     */
+    public function __construct(EncoderFactoryInterface $encoderFactory)
+    {
+        $this->encoderFactory = $encoderFactory;
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
We use lowercased `{@inheritdoc}` in whole Sylius, PhpStorm makes it harder to follow that convention :)